content/docs/add-react-to-a-website.md

```diff
@@ -1,35 +1,34 @@
 ---
 id: add-react-to-a-website
-title: Add React to a Website
+title: React को Website से जोड़े
 permalink: docs/add-react-to-a-website.html
 redirect_from:
   - "docs/add-react-to-an-existing-app.html"
 prev: getting-started.html
 next: create-a-new-react-app.html
 ---
+अपनी आवश्यकता के अनुसार कम या ज्यादा React का उपयोग करें
 
-Use as little or as much React as you need.
+क्रमिक गोद या अपने तरीके से उपयोग करने के लिए शुरू से ही इसी तरह डिज़ाइन की गई है, और **आप आवश्यकतानुसार कम या ज्यादा इसका उपयोग कर सकते हैं**। शायद आप केवल एक मौजूदा पेज पर कुछ "अन्तरक्रियाशीलता के छिड़काव( इसके अंतर्गत कुछ छोटा सा काम )" जोड़ना चाहते हैं। React Component ऐसा करने का एक शानदार तरीका है।
 
-React has been designed from the start for gradual adoption, and **you can use as little or as much React as you need**. Perhaps you only want to add some "sprinkles of interactivity" to an existing page. React components are a great way to do that.
-
-The majority of websites aren't, and don't need to be, single-page apps. With **a few lines of code and no build tooling**, try React in a small part of your website. You can then either gradually expand its presence, or keep it contained to a few dynamic widgets.
+अधिकांश वेबसाइटों को एकल-पृष्ठ ऐप्स होने की आवश्यकता नहीं है, और नहीं भी। **कोड की कुछ पंक्तियों और बिना किसी बिल्ड टूलिंग** के साथ, अपनी वेबसाइट के एक छोटे से हिस्से में React उपयोग  करने का प्रयास करें। आप तब या तो धीरे-धीरे अपनी उपस्थिति का विस्तार कर सकते हैं, या इसे कुछ गतिशील विजेट में समाहित में रखे।
 
 ---
 
-- [Add React in One Minute](#add-react-in-one-minute)
-- [Optional: Try React with JSX](#optional-try-react-with-jsx) (no bundler necessary!)
+- [एक मिनट में React जोड़ें](#add-react-in-one-minute)
+- [वैकल्पिक: JSX के साथ React का उपयोग करे](#optional-try-react-with-jsx) ( किसी भी बंडल आवश्यकता नहीं !)
```

Comment:
`( किसी भी बंडल आवश्यकता नहीं !)` => `(किसी भी बंडल आवश्यकता नहीं!)`

remove unwanted spaces

Reply:
`बंडल` => `बंडलर की`

Reply:
this is pending